_posts/2021-06-08-moveit-vs-moveit2.md

```diff
@@ -0,0 +1,46 @@
+categories:
+- ROS2
+- MoveIt
+---
+
+With the release of ROS 2 Galactic, we would like to share the new features only available in MoveIt 2. We hope to encourage the community to upgrade to MoveIt 2 as soon as possible. MoveIt 2 is out of beta.
+
+
+|                                                                                                                                                    | MoveIt 1 | MoveIt 2       |
+| -------------------------------------------------------------------------------------------------------------------------------------------------- | -------- | -------------- |
+| Motion Planning                                                                                                                                    | ✓        | ✓              |
+| Inverse Kinematics                                                                                                                                 | ✓        | ✓              |
+| Perception                                                                                                                                         | ✓        | ✓              |
+| Grasping                                                                                                                                           | ✓        | ✓              |
+| Setup Assistant                                                                                                                                    | ✓        | In Development |
+| Game controller integration for Servo                                                                                                              | ✓        | ✓              |
+| [Probabilistically complete Cartesian Planning](https://github.com/ros-planning/moveit2/tree/main/moveit_demo_nodes/run_ompl_constrained_planning) | ✕        | ✓              |
+| ROS 2 Component Nodes                                                                                                                              | ✕        | ✓              |
+| [Planning for differential drive bases](http://moveit2_tutorials.picknik.ai/doc/mobile_base_arm/mobile_base_arm_tutorial.html)                     | ✕        | ✓              |
+| [Hybrid planning](https://github.com/ros-planning/moveit2/pull/488) (global + local planners)                                                      | ✕        | ✓              |
+| Realtime support via DDS                                                                                                                           | ✕        | ✓              |
+| Improved cross-platform support                                                                                                                    | ✕        | ✓              |
+| Actively developed by PickNik & ROS 2 Technical Steering Committee                                                                                 | ✕        | ✓              |
```

Comment:
I realise this is mostly marketing material, but who from the TSC is actively helping ROS 2 development? Does @davetcoleman count?

Reply:
Maybe it should be broken into 2 categories and rephrased, like:

`Actively developed by PickNik`
`Development is coordinated by the TSC`

Reply:
Maybe just drop the TSC part or replace it with ROS 2 Manipulation Working Group.  Is any of the work done on MoveIt2 being coordinated by `the TSC`?

Reply:
Acutronic did, before they went bankrupt. Microsoft has collaborated with us a bit. We get some help from Open Robotics and various other companies on the TSC, like Amazon. All pretty limited admittedly.

Reply:
I think the bigger point here is that MoveIt 2 uses ROS 2 heavily, and ROS 2 is actively developed and maintained by the TSC.